en/scale-a-tidb-cluster.md

```diff
@@ -1,60 +1,74 @@
+Horizontally scaling TiDB means that you scale TiDB out or in by adding or remove Pods in your pool of resources. When you scale a TiDB cluster, PD, TiKV, and TiDB are scaled out or in sequentially according to the values of their replicas.
 
-Currently, the TiDB cluster supports management by TidbCluster Custom Resource (CR).
+- To scale out a TiDB cluster, **increase** the value of `replicas` of a certain component. The scaling out operations add Pods based on the Pod ID in ascending order, until the number of Pods equals the value of `replicas`.
 
-### Scale PD, TiDB, and TiKV
+- To scale in a TiDB cluster, **decrease** the value of `replicas` of a certain component. The scaling in operations remove Pods based on the Pod ID in descending order, until the number of Pods equals the value of `replicas`.
 
-Modify `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` in the `TidbCluster` object of the cluster to a desired value using kubectl. You can modify the values in the local file or using online command.
+### Horizontally scale PD, TiKV, and TiDB
 
-- You can also online modify the `TidbCluster` definition in the Kubernetes cluster by running the following command:
+To scale PD, TiKV, or TiDB horizontally, use kubectl to modify `spec.pd.replicas`, `spec.tikv.replicas`, and `spec.tidb.replicas` in the `TidbCluster` object of the cluster to a desired value. You can modify the values in the local file or using online command.
+
+1. Modify the `replicas` value of a component as needed. For example, running the following command can dynamically configure the `replicas` value of PD to 3:
 
     {{< copyable "shell-regular" >}}
 
     ```shell
-    kubectl edit tidbcluster ${cluster_name} -n ${namespace}
+    kubectl patch -n ${namespace} tc ${cluster_name} --type merge --patch '{"spec":{"pd":{"replicas":3}}}'
     ```
 
-Check whether the TiDB cluster in Kubernetes has updated to your desired definition by running the following command:
+2. Check whether your configuration has been updated in the corresponding TiDB cluster in Kubernetes.
 
-{{< copyable "shell-regular" >}}
+    {{< copyable "shell-regular" >}}
 
-```shell
-kubectl get tidbcluster ${cluster_name} -n ${namespace} -oyaml
-```
+    ```shell
+    kubectl get tidbcluster ${cluster_name} -n ${namespace} -oyaml
+    ```
 
-In the `TidbCluster` file output by the command above, if the values of `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` are consistent with the values you have modified, check whether the number of `TidbCluster` Pods has increased or decreased by running the following command:
+    In the `TidbCluster` file output by the command above, the values of `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` are expected to be consistent with the values you have configured.
```

Comment:
```suggestion
    In the `TidbCluster` CR output by the command above, the values of `spec.pd.replicas`, `spec.tidb.replicas`, and `spec.tikv.replicas` are expected to be consistent with the values you have configured.
```

Reply:
@DanielZhangQD Generally, we'll limit the use of the word `should` in documentation, so I'll describe this para in another way.